src/main/java/io/dropwizard/kafka/health/KafkaAdminHealthCheck.java

```diff
@@ -19,7 +20,7 @@ public KafkaAdminHealthCheck(final AdminClient adminClient, final String name) {
     }
 
     @Override
-    protected Result check() throws Exception {
```

Comment:
Never thrown

Reply:
The method signature is defined by the `HealthCheck` class